src/topology/sheaves/skyscraper.lean

```diff
@@ -140,8 +142,8 @@ colimit
 -/
 noncomputable def skyscraper_presheaf_cocone_is_colimit_of_not_specializes
   {y : X} (h : ¬p₀ ⤳ y) : is_colimit (skyscraper_presheaf_cocone p₀ A y) :=
-let h1 : ∃ (U : open_nhds y), p₀ ∉ U.1 :=
-  let ⟨U, ho, h₀, hy⟩ := not_specializes_iff_exists_open.mp h in ⟨⟨⟨U, ho⟩, h₀⟩, hy⟩ in
+let h0 := not_specializes_iff_exists_open.mp h, h1 : ∃ (U : open_nhds y), p₀ ∉ U.1 :=
+  ⟨⟨⟨h0.some, h0.some_spec.1⟩, h0.some_spec.2.1⟩, h0.some_spec.2.2⟩ in
```

Comment:
Why revert this? Doesn't seem to cause any problems.

Reply:
I must forgot to `pull` before commit

Reply:
This has been fixed